tests/Issues/AutoImport/Fixture/DocBlock/two_routes_with_comment_before.php.inc

```diff
@@ -26,7 +26,6 @@ use Symfony\Component\Routing\Annotation\Route;
 class TwoRoutesWithCommentBefore
 {
     /**
-     * Testsssssssssss
```

Comment:
I am temporary removing this comment due to the keep comment feature cause a bug on PR:

- https://github.com/rectorphp/rector-src/pull/5240

when rebased, that cause invalid change on nested annotation:

```diff
1) Rector\Core\Tests\Issues\FqcnAnnotationToAttribute\FqcnAnnotationToAttributeTest::test with data set #0
Failed on fixture file "fixture.php.inc"
Failed asserting that string matches format description.
--- Expected
+++ Actual
@@ @@
 
 namespace Rector\Core\Tests\Issues\FqcnAnnotationToAttribute\Fixture;
 
+/**
+ *
+@\Doctrine\ORM\Mapping\Table("user", indexes={
+ ,
+ ,
+})
+*/
 #[\Doctrine\ORM\Mapping\Entity]
```

I will check if it can be resolved there instead.